docs/new_docs/elter_model/temportalCoverages.md

```diff
@@ -25,6 +26,29 @@
         }
       }
     }
+  },
+  "temporalResolution": {
```

Comment:
@maab Isn't this more of file metadata attribute? Because I can imagine datasets containing multiple files with different temporal resolutions.

Reply:
It should indicate a general/overall temporal resolution of the dataset. If the dataset contains more then one file the lowest frequency should be reported.